### PR TITLE
TEC-3464 - DQA Fixes

### PR DESCRIPTION
**Pull request by jesseeproductions**
DQA Fixes
-Style Updates
-Processed the SVG through svgomg

### CODE REVIEW

File: src/resources/images/alert.svg [hunk 1]
```diff
@@ -1 +1,7 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18"><defs/><g fill="none" fill-rule="evenodd" transform="translate(1.533333 1.533333)"><circle cx="7.46666667" cy="7.46666667" r="7.46666667" stroke="#DA394D" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5"/><circle cx="7.46666667" cy="11.2" r="1.06666667" fill="#DA394D" fill-rule="nonzero"/><path stroke="#DA394D" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M7.46666667 3.2V8"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```
Comment (Camwyn) [inline src/resources/images/alert.svg:1]: do we need to SVGO (minify) this?

File: src/admin-views/metabox/compatibility/events-control-extension/mark-online.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
  */
 
 ?>
-<div class="tribe-events-control-metabox-container" style="margin-top: 24px;">
+<div class="tribe-events-control-metabox-container" style="margin-top: 18px;">
```
Comment (Camwyn) [inline src/admin-views/metabox/compatibility/events-control-extension/mark-online.php:25]: Does this have to be an inline style?

**Review by Camwyn — APPROVED** [review]
Approval with two questions - if either can be addressed, great! If not, merge.

File: src/admin-views/metabox/compatibility/events-control-extension/mark-online.php (hunk 2 above)
Comment (jesseeproductions) [reply src/admin-views/metabox/compatibility/events-control-extension/mark-online.php:25]: It is compatibility coding and will be removed one day so I'd rather not add an entry in the css.

File: src/resources/images/alert.svg (hunk 1 above)
Comment (jesseeproductions) [reply src/resources/images/alert.svg:1]: I did run it through this site: https://jakearchibald.github.io/svgomg/

Comment (Camwyn) [reply src/resources/images/alert.svg:1]: weird, ok

**Pull request merged**